priv/base_prototypes.json

```diff
@@ -2,19 +2,15 @@
   "entities": [
     {
       "id": "_player",
-      "behaviors": ["visible", "player"],
-      "props": {
-        "description": "is a generic player."
-      }
+      "behaviors": [
+        { "type": "visible", "description": "is a generic player." },
+        "player"
+      ]
     },
     {
       "id": "_god",
       "proto": "_player",
       "behaviors": ["god"]
```

Comment:
don't we need to update this behavior as well?

Reply:
[`Marshal.resolve_behavior`](https://github.com/dorodango/doro/blob/a2290932879f05e38b0ed3c5ce27b5e08b81d62a/lib/doro/world/marshal.ex#L62) has a couple of function heads to handle simple strings.  I think we will probably have a handful of behaviors that have no properties.  Although I'm not sure if `unresolve_behaviors` does the right thing...